src/04-modality-specific-files/04-intracranial-electroencephalography.md

```diff
@@ -343,6 +365,16 @@ MUST be present **in this specific order**:
 | z               | REQUIRED              | Z position. If electrodes are in 2D space this should be a column of `n/a` values.                                               |
 | size            | REQUIRED              | Surface area of the electrode, units MUST be in `mm^2`.                                                                          |
 
+{{ MACROS___make_columns_table(
+   {
+      "name_electrodes": "REQUIRED",
+      "x": "REQUIRED",
+      "y": "REQUIRED",
+      "z": "REQUIRED",
```

Comment:
iEEG can contain 2D elec locations, ...originally the spec had a comment about that here, that `z` may be fully `n/a`. I think it'd be good to keep that info.